RealmSwift/AsymmetricObject.swift

```diff
@@ -85,6 +85,40 @@ extension AsymmetricObject {
         return ObjectUtil.getSwiftProperties(self)
     }
 
+    // MARK: Object Customization
+
+    /**
+     Override this method to specify a map of public-private property names.
+     This will set a different property column name on the Realm, and allows using the public name
+     for any operation with the property. (Ex: Queries, Sorting, ...).
+     This very helpful if you need to map property names from your `Device Sync` JSON schema
+     to local property names.
```

Comment:
I think we want to avoid using the word "column" in anything public. .NET mentions a "persisted property name", while JS uses "underlying".